example/helloWorld/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+project(alpaka-helloworld)
+cmake_minimum_required(VERSION 3.0.1)
+
+################################################################################
+# Find alpaka
+################################################################################
+SET(ALPAKA_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/../../" CACHE STRING  "The location of the alpaka library")
+LIST(APPEND CMAKE_MODULE_PATH "${ALPAKA_ROOT}")
+
+find_package("alpaka" REQUIRED)
+include_directories(SYSTEM ${alpaka_INCLUDE_DIRS})
+add_definitions(${alpaka_DEFINITIONS})
+ADD_DEFINITIONS(${alpaka_DEFINITIONS} ${ALPAKA_DEV_COMPILE_OPTIONS})
+set(LIBS ${LIBS} ${alpaka_LIBRARIES})
+
+
+################################################################################
+# Compiler Flags
+################################################################################
+# GNU
+if(CMAKE_COMPILER_IS_GNUCXX)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wextra")
```

Comment:
These flags (Wall, Wextra) are part of `${ALPAKA_DEV_COMPILE_OPTIONS}` as well as others for other platforms.
Therefore you can remove this part.